### PR TITLE
CB-1095 Add stub rdb service

### PR DESCRIPTION
**Pull request by bhavanki**
A new service called "rdb" is introduced, for the eventual purpose of
managing database servers and databases. It is currently a stub and does
nothing, but builds successfully as part of the overall Cloudbreak build
process.

### CODE REVIEW

**Comment by bhavanki** [comment]
The Spotbugs CircleCI test seems to have failed because there isn't enough memory available (https://stackoverflow.com/questions/38967991/why-are-my-gradle-builds-dying-with-exit-code-137). Everything passes locally for me.

File: rdb/src/main/java/com/sequenceiq/rdb/controller/mapper/WebApplicaitonExceptionMapper.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.sequenceiq.rdb.controller.mapper;
+
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+public class WebApplicaitonExceptionMapper implements ExceptionMapper<WebApplicationException> {
```
Comment (gergopapi2) [inline rdb/src/main/java/com/sequenceiq/rdb/controller/mapper/WebApplicaitonExceptionMapper.java]: Typo WebApplicaiton => WebApplication

Comment (bhavanki) [reply rdb/src/main/java/com/sequenceiq/rdb/controller/mapper/WebApplicaitonExceptionMapper.java]: Nice find :)

**Comment by bhavanki** [comment]
The Spotbugs test failed again, most likely due to a low memory condition. However, it passes locally, so I'll merge this. I'll ping the team about getting CircleCI fixed.

**Pull request merged**